src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -41,7 +41,7 @@
     HEREDOC
   end
 
-  describe 'POST #submit_request' do
+  describe 'POST #create_submit_request' do
```

Comment:
These controller specs need to be moved to a new file for the new `submissions_controller`.

Reply:
I'm on it

Reply:
All the remaining failing controller specs are related to this. I'm taking care of writing some feature specs since there are none.

Reply:
The specs are moved